vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -66,6 +66,15 @@ import { extractDateParts, parseDate as _parseDate } from '@vaadin/date-picker/s
           }
 
           function isShortYearFormat(format) {
+            // Format is considered short if it includes year
+            // as "yy" or "YY", or no year specified at all.
+            if (!format.includes('y') && !format.includes('Y')) {
+              return true;
+            }
+            // Format with days only is also considered short.
+            if (!format.includes('m') && !format.includes('M')) {
```

Comment:
I wonder if the previous check already covers this case, unless there is a possibility of `DD.YY` format.

Reply:
It doesn't cover it because the `dd` format is then not considered short and here's what happens:

```
 ❌ date-picker connector > dd format > should parse date using dd format
      AssertionError: expected { day: 1, month: 1, year: 2024 } to deeply equal { day: 1, month: 0, year: 2024 }
      + expected - actual

       {
         "day": 1
      -  "month": 1
      +  "month": 0
         "year": 2024
       }
```

Reply:
Oh, I missed this, thanks